back/app/services/idea_custom_fields_service.rb

```diff
@@ -8,7 +8,7 @@ def initialize(custom_form)
 
   def all_fields
     if @custom_form.custom_field_ids.empty?
-      participation_method.default_fields custom_form
+      @participation_method.default_fields @custom_form
```

Comment:
This was the first thing I changed as I thought it could be the issue, but turned out not to be - left it though as thought it should be referring to instance variables, but has only not been caught as an issue because I don't think it's possible to have a custom form without any custom fields? Please correct if I'm wrong

Reply:
Since we have attribute accessors for participation_method and custom_form, it would be better to use those and remove the "@" everywhere. Alternatively, we could remove the attribute accessors.

I don't have a strong preference, as long as we keep things consistent. I think Koen DH used to have a strong preference for attribute accessors.